pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -216,6 +237,53 @@ const New = () => {
     setFormErrors({})
     setForm({ ...form, [propertyName]: value })
   }
+
+  const handleMultiForm = ({ propertyName, value }) => {
+    setMultiFormErrors({})
+    setMultiChoiceForm({ ...multiChoiceForm, [propertyName]: value })
+  }
+
+  const handleNotaButton = () => {
+    const options = multiChoiceForm.options;
+    const last = options.length - 1;
+
+    if (options[last] !== "None of the Above") {
+      options.push("None of the Above");
+      handleMultiForm({propertyName: "options", value: options});
+      setHideNotaButton(true);
+    }
+  }
+
+  const updateOptions = (odx: number, value: string, addOption: boolean)  => {
```

Comment:
let's have an `updateOption` function and a `addOption` function instead of making one function with a weird API. I'm quite confused what this `odx === -1` stuff is and it should probably not exist.

Reply:
Done, separated addition, removal and update into three different functions.